js/components/PlaylistWindow/ScrollBar.tsx

```diff
@@ -1,16 +1,25 @@
 import React from "react";
 import { connect } from "react-redux";
-import Slider from "rc-slider/lib/Slider";
+import Slider from "rc-slider";
```

Comment:
Looks like the Typescript types don't include the deep/partial import. The top-level import works, but it bring along a ton of extra stuff. Any idea how I can tell TypeScript to use the types from the top level import? @durasj

Reply:
They do work in the typescript but the problem is with the types. You can see in the package that they only include a index.ts in the root and therefore typescript doesn't know how to type it. You could let typescript ignore that line and type the import on the next line by importing just the interface / type front the types.

Reply:
I guess it's not very clear what I mean so I can show you what I mean later if it'll won't be clear to you.

Reply:
That should be pretty simple. What I had in mind was something like:

```js
import RcSlider from "rc-slider";
// @ts-ignore
import SliderComponent from "rc-slider/lib/Slider";
const Slider = SliderComponent as RcSlider;
```

Typescript should be smart enough to use only the types. But that doesn't look that good. So the Right Way would be to either make a PR in the DefinitelyTyped repo with `slider.d.ts` placed in the `@types/rc-slider/lib` directory and content:

```js
import Slider from '../index';
export default Slider;
```

Or you can declare it yourself locally (untill your PR gets accepted?) just like you have the `static.d.ts`.

Reply:
Thanks! That seems to have worked.